packages/rke2-flannel/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,12 +1,16 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -3,7 +3,7 @@
+@@ -1,9 +1,9 @@
+ apiVersion: v1
+-appVersion: v0.25.0-dirty
```

Comment:
I guess this needs to be changed if we remoed `-dirty` in the upstream chart?

Reply:
It should be already fixed

Reply:
done!